### ISSUE TITLE
By default, the mouse speed is too slow

### ISSUE THREAD

**Issue by goupildb** [mgerhardy/caveexpress#106]
Sorry for the harassment, this is my last suggestion :)

By default, the mouse speed is too slow.
It is very painful to go to the menu (Settings ➜ Input ➜ mouse Speed ➜ and a lot of mouse movements to reach the cursor) to adjust its speed.
I suggest you this default setting
INFO: (LOG_COMMON): mousespeed => changed value to: 2.000000

**Comment by mgerhardy** [comment]
> Sorry for the harassment, this is my last suggestion :)

Hopefully not - very much appreciated. To be honest the games were designed to be controlled by the keyboard.

**Comment by mgerhardy** [comment]
For me 2.0 is way too fast. 0.2 is what my desktop uses, too. So there must be some bug and only fixing the value for you wouldn't solve the real issue. Can you give me the full log that the game produces?

**Comment by goupildb** [comment]
Ok, i have understand, it's because he had stored its parameters on 
 ~/.local/share/caveexpress/caveexpress.sqlite
What is surprising is that the mouse cursor is displayed in its minimum value while it is actually at the last value recorded when leaving the game

**Comment by goupildb** [comment]
> To be honest the games were designed to be controlled by the keyboard.

I agree with you, and the game is easily controlled by keyboard and joystick, but as we see a mouse cursor, we tend to want to use it :)

**Comment by cryham** [comment]
Likely solved already.
I just tried latest version (I got built from git repo) and it is okay. I would still make it faster (maybe it depends on screen resolution?).
But current version I got installed just from Debian 12 repositories, was indeed horribly slow.

**Comment by cryham** [comment]
Okay just tested CavePacker got now from Debian 12 repositories.
This is a serious issue (maybe fixed in git already?). I couldn't move mouse anywhere, only like in top left 10% of screen 20% max. I think this is maybe because of mouse grab? Since I can't just move infinitely, it hits screen border, and at much smaller wrong factor.
How can one set mouse speed in GUI only using keyboard?
I had to edit /home/user/.local/share/cavepacker/cavepacker.sqlite
and mousespeed was 0.2 there, I set it to 2.8 which is max slider, should be IMO the default.

**Comment by mgerhardy** [comment]
10244b2e674fab43c7103bacc2f75e3006ab1337 this is now using the system mouse cursor.
